_includes/notes_graph.html

```diff
@@ -46,10 +46,15 @@
     const FONT_BASELINE = 15;
     const MAX_LABEL_LENGTH = 50;
 
-    const graphData = {% include notes_graph.json %}
+    const graphData = {% include tagged_notes_graph.json %}
+    const tag = `{{ include.tag }}`
+    const pageTag = tag.split(" ")[0]
 
-    let nodesData = graphData.nodes;
-    let linksData = graphData.edges;
+    if(tag != "") {
+
+
+    let nodesData = graphData.nodes[`${pageTag}`];
+    let linksData = graphData.edges[`${pageTag}`];
```

Comment:
Need to pull in tag dynamically into the include file. 

TODO: handle multiple tags. This will break if you have more than one tag for a page (it will assume it's a different tag) as multiple tags get passed in as a concatinated string